Yasgui/packages/yasgui/src/index.ts

```diff
@@ -292,21 +294,87 @@ export class Yasgui extends EventEmitter {
     }
     return tab;
   }
-  public closeOtherTabs(currentTabId: string): void {
+  public closeOtherTabs(skipTabId: string): void {
```

Comment:
Revert this!

Reply:
I renamed it because the name 'currentTabId' is a little confusing. The passed argument is not 'currentTabId'; it is the ID of the tab when the event is fired. The event can be fired from any tab, not only the current one

Reply:
I know, but as we already agreed, we try to avoid unnecessary changes in the yasgui source